client/app/resources/i18n/hosting/Messages_fr_FR.xml

```diff
@@ -1408,6 +1510,6 @@
    <translation id="hosting_change_main_domain_warning_delete_emails" qtlid="286565">Attention ! La suppression de votre service e-mail existant supprimera toutes les adresses e-mail actuellement créées avec votre nom de domaine.</translation>
 
    <translation id="hosting_dashboard_general_informations" qtlid="27115">Informations générales</translation>
-   <translation id="hosting_dashboard_configuration" qtlid="4406">Configuration</translation>
+   <translation id="hosting_dashboard_configuration" qtlid="4406">Personnalisation</translation>
```

Comment:
Please make a new translation entry.
Right now this will replace every instance of "Configuration" with "Personnalisation" **everywhere** in the manager.

Reply:
Also, I would call the **whole feature** "Runtime" instead of "Configuration", because it's about installing and configuring runtimes. I would change the tab name to "Runtime" or "Environnements d'exécution" in French.

Reply:
Again, it's a call made from UX/Content during a certain meeting, I am not able to answer this request before approval (but yeah, I will definitevely make a new translation entry btw)

Reply:
I got the confirmation from @jleveugle that "Runtime" would be a better name.